### PR TITLE
Add script to transfer items from one container to another.

### PR DESCRIPTION
**Pull request by Sarvatt**

### CODE REVIEW

**Comment by Sarvatt** [comment]
I guess common isn't even needed here

File: transfer-items.lic [hunk 1]
```diff
@@ -0,0 +1,48 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#transfer-items
+=end
+
+custom_require.call(%w[common])
```
Comment (rpherbig) [inline transfer-items.lic]: Is this needed?

File: transfer-items.lic [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+class ItemTransfer
+  include DRC
```
Comment (rpherbig) [inline transfer-items.lic]: Is this needed?

File: transfer-items.lic [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    fput("look in my #{@source}")
+
+    while line = get
+      if line =~ /In the .*#{@source} you see/
+        items = line
+        break
+      end
+    end
+
+    items = items.sub(' and ', ',')
```
Comment (rpherbig) [inline transfer-items.lic]: What do you think about something like this? (in place of lines 34-44)

```
    items
      .sub(' and ', ',')
      .split(',')
      .map { |full_name| full_name.split(' ') }
      .map { |parts| parts.last.sub('.', '') }
      .each do |item|
        fput("get #{item} from my #{@source}")
        fput("put #{item} in my #{@destination}")
      end
```

Comment (Sarvatt) [reply transfer-items.lic]: that works great actually, thank you for the nicer way to do it

**Comment by rpherbig** [comment]
Please update https://elanthipedia.play.net/Lich_script_repository#transfer-items when you get a chance

**Comment by Sarvatt** [comment]
done, thank you

**Pull request merged**